### PR TITLE
Release 1.100.2

### PR DESCRIPTION
**Pull request by geriux**
Release for Gutenberg Mobile 1.100.2, this will be integrated only for the iOS host app.

## Related PRs

- Gutenberg: https://github.com/WordPress/gutenberg/pull/53293
- WPAndroid: N/A
- WPiOS: https://github.com/wordpress-mobile/WordPress-iOS/pull/21239

- Aztec-iOS: N/A
- Aztec-Android: N/A

## Extra PRs that Landed After the Release Was Cut

No extra PRs yet. 🎉

### Fix iOS Focus loop for RichText components
- **PR:** https://github.com/WordPress/gutenberg/pull/53217
- **Issue:** https://github.com/wordpress-mobile/gutenberg-mobile/issues/6002

## Test plan

Once the installable builds of the main apps are ready, perform a quick smoke test of the editor on both iOS and Android to verify it launches without crashing. We will perform additional testing after the main apps cut their releases.

## Release Submission Checklist

- [x] Verify Items from test plan have been completed
- [x] Check if `RELEASE-NOTES.txt` is updated with all the changes that made it to the release. Replace `Unreleased` section with the release version and create a new `Unreleased` section.
- [x] Check if `gutenberg/packages/react-native-editor/CHANGELOG.md` is updated with all the changes that made it to the release. Replace `## Unreleased` with the release version and create a new `## Unreleased`.
- [x] Bundle package of the release is updated.

### CODE REVIEW

**Comment by geriux** [comment]
## 🟢 Writing Flow checklist (iOS)
**Build**: [jetpack-installable-builds #5625](https://install.appcenter.ms/orgs/automattic/apps/jetpack-installable-builds/releases/5625)
**Device**: iPhone 14 Pro (16.5.1 (c))

#### General
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/copy-paste.md#tc001) - Paste formatted text copied from website
#### Rich Text Format
- [x] [TC007](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc007) - Test format detection under the cursor
- [x] [TC009](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc009) - Test autocorrection doesn't apply formatting to Heading
- [x] [TC010](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/rich-text-formatting.md#tc010) - Test autocorrection doesn't remove formatting from Heading
##### Splitting and merging
- [x] [TC001](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc001) - Merge after writing
  - [x] Paragraph
  - [x] Heading
- [x] [TC002](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc002) - Merge after selection
  - [x] Paragraph
  - [x] Heading
- [x] [TC003](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc003) - Merge after deleting text
  - [x] Paragraph
  - [x] Heading
- [x] [TC004](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc004) - Merge after deleting all
  - [x] Paragraph
  - [x] Heading
- [x] [TC005](https://github.com/wordpress-mobile/test-cases/blob/trunk/test-cases/gutenberg/writing-flow/splitting-merging.md#tc005) - Merge multiple blocks
  - [x] Paragraph
  - [x] Heading

**Comment by geriux** [comment]
## 🟢 New changes (iOS)
**Build**: [jetpack-installable-builds #5625](https://install.appcenter.ms/orgs/automattic/apps/jetpack-installable-builds/releases/5625)
**Device**: iPhone 14 Pro (16.5.1 (c))

- [x] https://github.com/WordPress/gutenberg/pull/53217

**Review by SiobhyB — APPROVED** [review]
Approved via https://github.com/WordPress/gutenberg/pull/53293#pullrequestreview-1561068401

File: src/test/index.js [hunk 1]
```diff
@@ -80,6 +80,5 @@ describe( 'Gutenberg Mobile initialization', () => {
 		const blockList = screen.getAllByTestId( 'block-list-wrapper' )[ 0 ];
 
 		expect( blockList ).toBeVisible();
-		expect( console ).toHaveLoggedWith( 'Hermes is: true' );
```
Comment (geriux) [inline src/test/index.js:83]: Hey @dcalhoun 👋 does this change look correct? I was having CI failures and saw that this https://github.com/WordPress/gutenberg/pull/53268 change was recently merged. It looks like we shouldn't assert that this was logged anymore. Is it accurate? Let me know, thank you!

Comment (dcalhoun) [reply src/test/index.js:83]: Yes, you are correct. This assertion needs removal. Thank you for taking care of this.

Comment (geriux) [reply src/test/index.js:83]: Awesome! No problem 👍 Thanks!

**Comment by geriux** [comment]
FYI I'll merge with the failing `Build iOS RN XCFramework & Publish to S3` job since we aren't referencing this commit anywhere. Hopefully, a fix will land in `trunk` soon.

**Pull request merged**